### PR TITLE
Pages errors

### PR DESCRIPTION
**Pull request by mapkyca**
## Here's what I fixed or added:

Removed \Exception type constraint.

Removed a notice error.

## Here's why I did it:

PHP7 has different type hierarchy.

### CODE REVIEW

File: IdnoPlugins/StaticPages/templates/default/staticpages/admin.tpl.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
 <div class="row">
     <div class="col-md-10 col-md-offset-1">
         <p class="pages">
-            <a href="<?= \Idno\Core\Idno::site()->config()->getDisplayURL() ?>staticpages/edit/?category=<?= urlencode($category) ?>"
+            <a href="<?= \Idno\Core\Idno::site()->config()->getDisplayURL() ?>staticpages/edit/"
```
Comment (kylewm) [inline IdnoPlugins/StaticPages/templates/default/staticpages/admin.tpl.php]: is this related?

Comment (physcocode) [inline IdnoPlugins/StaticPages/templates/default/staticpages/admin.tpl.php]: I was gonna ask the same

Comment (mapkyca) [inline IdnoPlugins/StaticPages/templates/default/staticpages/admin.tpl.php]: No, but it was causing a notice error (and therefore an exception, since I run with pedantic mode on). Needed to tidy it up first, and didn't see much point in making this a second PR

Comment (kylewm) [inline IdnoPlugins/StaticPages/templates/default/staticpages/admin.tpl.php]: 👍 just wanted to make sure it was on purpose

**Comment by rosetree** [comment]
Please excuse my late reply. I can confirm that the commit cf3bec8269e5efa6313deb128c23fe34a467737d resolves my issue #1477. Thank you, @mapkyca!

**Comment by mapkyca** [comment]
HTH :)
